src/components/OrganizationSwitch.tsx

```diff
@@ -99,16 +99,16 @@ const OrganizationSwitch = () => {
           <span>Select Organization</span>
           <ul>
             <li>
-              <Link to="/">username</Link>
+              <Link to="/">{state.user.username ?? "unknown"}</Link>
             </li>
-            {state.fakeOrganizations.map((organization) => (
+            {state.user.orgs?.map((organization) => (
```

Comment:
real orgs. i have these nested under user, but they don't really have to be there...